releasenotes/notes/cmd-jmx-options-6a1a75d4bbf62ca2.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
```

this is really a bugfix IMO, so should be under `fixes`:

```suggestion
fixes:
```

---

Done